### PR TITLE
AQCU-1619: Dockerfile and other parts

### PR DESCRIPTION
**Pull request by mhines-usgs**

### CODE REVIEW

**Comment by isuftin** [comment]
Shall we hook Travis up to this?

**Comment by mhines-usgs** [comment]
Sure, you want to? I'm still making sure this works locally. :whack-a-mole:

**Comment by isuftin** [comment]
Done

**Comment by isuftin** [comment]
You will need to add a `.travis.yml` if you do want this to be built properly

**Comment by mhines-usgs** [comment]
The compose for this still pukes on my windows machine, but, so does the one for the gateway. I don't think it's written for it to work on windows/docker toolbox.

**Comment by mhines-usgs** [comment]
not sure exactly how to set the rserveHost here. 
https://github.com/USGS-CIDA/docker-aqcu-java-to-r/pull/1/commits/8595ed54b60c969767540cd4e4546807543b1a5b

File: rservePassword.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+rserve
```
Comment (zmoore-usgs) [inline rservePassword.txt]: I think this file needs to be nested under a /secrets directory. The compose file tries to mount it from `./secrets/rservePassword.txt`

**Comment by mhines-usgs** [comment]
Thanks, I was wondering about that after looking at the other docker repos. I also do think things will fail because the latest jar artifact doesn't have the other vars related to rserve.

**Pull request merged**